testgres/node.py

```diff
@@ -856,7 +863,24 @@ def backup(self, **kwargs):
 
         return NodeBackup(node=self, **kwargs)
 
-    def replicate(self, name=None, **kwargs):
+    def create_replication_slot(self, slot_name, dbname=None, username=None):
+        """
+        Create a physical replication slot.
+
+        Args:
+            slot_name: slot name
+            dbname: database name
+            username: database user name
+        """
+        query = (
+            "select pg_create_physical_replication_slot('{}')"
+        ).format(slot_name)
+
+        self.execute(query=query,
+                     dbname=dbname or default_dbname(),
```

Comment:
You don't have to explicitly use `default_dbname()` and such since `execute()` method is able to handle `None` args properly.

Reply:
Agree